obspy/geodetics/base.py

```diff
@@ -315,37 +315,42 @@ def locations2degrees(lat1, long1, lat2, long2):
     Earth. For more accurate values use the geodesic distance calculations of
     geopy (https://github.com/geopy/geopy).
 
-    :type lat1: float
-    :param lat1: Latitude of point 1 in degrees
-    :type long1: float
-    :param long1: Longitude of point 1 in degrees
-    :type lat2: float
-    :param lat2: Latitude of point 2 in degrees
-    :type long2: float
-    :param long2: Longitude of point 2 in degrees
-    :rtype: float
-    :return: Distance in degrees as a floating point number.
+    :type lat1: ``float or :class:`numpy.ndarray```
```

Comment:
I wrote it like that following your previous suggestion which had double backticks, I am confused but I assume they where intended for the formatting here on the page but they were rendered unformatted.
Whatever, hope to have understood what you mean, let me know